### DISCUSSION TITLE
#define ST7789_DRIVER shows a white screen but ST7789_2_DRIVER does not

### DISCUSSION THREAD

**Discussion by mike-rankin** [Bodmer/TFT_eSPI#2872] [Q&A - General]
Any idea what the differences are between ST7789_DRIVER and ST7789_2_DRIVER might be and why ST7789_DRIVER  shows a white screen? I'm likely missing a configuration or gpio causing the issue.

**Comment by Bodmer** [comment]
Some of the ST7789 screens need a different setup. This may be due to variations in the display panel. This was discovered by a user of the library.
